core/idotmatrix/common.py

```diff
@@ -36,6 +36,28 @@ def rotate180degrees(self, type=0):
         except BaseException as error:
             logging.error("could not rotate the screen of the device: {}".format(error))
 
+    def set_screen_brightness(self, brightness_percent: int) -> None:
```

Comment:
Nice catch with the type casting.. have missed that for the initial version. Will add this to all functions later on :)

Reply:
Yeah, type casting would be good practice and simplify the code :)